src/tables/cff.mjs

```diff
@@ -308,6 +308,17 @@ function interpretDict(dict, meta, strings) {
             if (m.type === 'SID') {
                 value = getCFFString(strings, value);
             }
+            if (m.type === 'delta' && value !== null) {
+                if (!Array.isArray(value) || value.length % 2 != 0) {
```

Comment:
I fact I can not see in the spec that an odd number of values is forbidden. But of course it would not make any sense, a BlueZone that starts somewhere and ends nowhere? I believe I remember to have read sometime that there must always be an even number of numbers grr, an even amount of numbers; but I can not find a reference for that.

Maybe someone knows more and/or you like to remove the even/odd test. It's in princible only a safeguard against unusable delta arrays. :woman_shrugging: